pages/common/stow.md

```diff
@@ -0,0 +1,24 @@
+# stow
+
+> Symlink manager.
+> Often used to manage dotfiles.
+
+- Symlink all files recursively to {{target}}:
+
+`stow --target={{target}} {{file1 folder1 file2 folder2}}`
+
+- Delete symlinks recursively from {{target}}:
+
+`stow --delete --target={{target}} {{file1 folder1 file2 folder2}}`
+
+- Simulate to see what the result would be like:
+
+`stow  --simulate --target={{target}} {{file1 folder1 file2 folder2}}`
```

Comment:
multiple spaces between stow and --simulate

Reply:
Thx for your catchy eye! :)